nms/server/host/routes.ts

```diff
@@ -168,7 +171,7 @@ router.post(
         ),
       });
       if (organization) {
-        return res.status(404).send({error: 'Organization already exists'});
+        return res.status(409).send({error: 'Organization already exists'});
```

Comment:
_Optional:_ As rephrasing was done above already, could we have `Organization exists already` here please.